frontend/app/tests/unit/components/settings/account-balances.spec.ts

```diff
@@ -12,7 +12,7 @@ import '../../i18n';
 Vue.use(Vuetify);
 
 describe('AccountBalances.vue', () => {
-  let wrapper: Wrapper<AccountBalances>;
+  let wrapper: Wrapper<any>;
```

Comment:
At some point we need to figure out how to properly type the composition API components, but probably after migrating to vue 3